### PR TITLE
added logic for greeting based on local time in navbar

### PR DESCRIPTION
**Pull request by rifkiridha**

### CODE REVIEW

File: src/components/organisms/navbar.jsx/index.jsx [hunk 1]
```diff
@@ -27,6 +29,23 @@ export default function Navbar() {
     }
   }, []);
 
+  useEffect(() => {
+    const currentTime = new Date().getHours();
+    let newGreeting = "";
+
+    if (currentTime >= 5 && currentTime < 12) {
+      newGreeting = "Pagi";
```
Comment (faisalrahmatdani) [inline src/components/organisms/navbar.jsx/index.jsx:37]: langsung aja setGreeting('Pagi gausah di taruh variable')

File: src/components/organisms/navbar.jsx/index.jsx [hunk 2]
```diff
@@ -27,6 +29,23 @@ export default function Navbar() {
     }
   }, []);
 
+  useEffect(() => {
+    const currentTime = new Date().getHours();
+    let newGreeting = "";
+
+    if (currentTime >= 5 && currentTime < 12) {
```
Comment (faisalrahmatdani) [inline src/components/organisms/navbar.jsx/index.jsx:36]: pake Switch Case aja

**Review by faisalrahmatdani — CHANGES_REQUESTED** [review]

File: src/components/organisms/navbar.jsx/index.jsx (hunk 1 above)
Comment (rifkiridha) [reply src/components/organisms/navbar.jsx/index.jsx]: done

**Review by faisalrahmatdani — APPROVED** [review]

File: src/components/organisms/navbar.jsx/index.jsx (hunk 2 above)
Comment (rifkiridha) [reply src/components/organisms/navbar.jsx/index.jsx]: pakai if aja wkwk

**Pull request merged**